mruby build errors, sometimes

There is something really flaky going on with the mruby build.  When it fails,
```
  bison -o "C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/y.tab.c" "C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/parse.y"
  bison -o "C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/y.tab.c" "C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/parse.y"
  bison -o "C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/y.tab.c" "C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/parse.y"
  clang -std=gnu99 -march=x86-64 -mtune=generic -O2 -pipe -DMRBGEM_MRUBY_COMPILER_VERSION=0.0.0 -I"C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/include" -I"C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core" -MMD -o "C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/build/host/mrbgems/mruby-compiler/core/y.tab.o" -c "C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/y.tab.c"
  C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/y.tab.c:70:9: warning: missing terminating '"' character [-Winvalid-pp-token]
  #line 7 "mrbgems/mruby-compiler/core/parse.y
          ^
  C:/_/M/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/y.tab.c:70:9: error: invalid filename for #line directive
...
```

When it succeeds, `bison` is not invoked.
```
  clang -std=gnu99 -march=x86-64 -mtune=generic -O2 -pipe -DMRBGEM_MRUBY_COMPILER_VERSION=0.0.0 -I"C:/_/mingw-w64-mruby/src/build-x86_64-w64-mingw32/include" -I"C:/_/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core" -MMD -o "C:/_/mingw-w64-mruby/src/build-x86_64-w64-mingw32/build/host/mrbgems/mruby-compiler/core/y.tab.o" -c "C:/_/mingw-w64-mruby/src/build-x86_64-w64-mingw32/mrbgems/mruby-compiler/core/y.tab.c"
```
I have not been able to determine when bison is run and when it is not

--- a/mingw-w64-mruby/PKGBUILD
+++ b/mingw-w64-mruby/PKGBUILD
@@ -4,7 +4,7 @@ _realname=mruby
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=2.1.2
-pkgrel=3
+pkgrel=4
 pkgdesc="mruby is the lightweight implementation of the Ruby language (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
@@ -19,6 +19,9 @@ build() {
   [[ -d ${srcdir}/build-${MINGW_CHOST} ]] && rm -rf ${srcdir}/build-${MINGW_CHOST}
   cp -rf "${srcdir}/${_realname}-${pkgver}" ${srcdir}/build-${MINGW_CHOST}
   cd ${srcdir}/build-${MINGW_CHOST}
+  # make sure that generated file is newer
+  sleep 1
+  touch mrbgems/mruby-compiler/core/y.tab.c
 
   ./minirake -v -j1
 }

